doc/api_assets/style.css

```diff
@@ -16,7 +16,9 @@ body {
 }
 
 pre, tt, code, .pre, span.type, a.type {
-  font-family: Monaco, Consolas, "Lucida Console", monospace;
+  font-size: 1.2em;
```

Comment:
Not a fan of this size definition here, it will cause all sorts of problems. Better to set it on just the code blocks if you feel they're too small , e.g. `code[class*="language-"], pre[class*="language-"]`, but I'd prefer not to mess with font sizes at all for inline elements.